r/geoarrow/R/arrow-compat.R

```diff
@@ -0,0 +1,177 @@
+
+as_arrow_array.geoarrow_vctr <- function(x, ..., type = NULL) {
+  chunked <- as_chunked_array.geoarrow_vctr(x, ..., type = type)
+  if (chunked$num_chunks == 1) {
```

Comment:
Does `arrow::as_array_array()` copy if we have a single chunk and we're avoiding that?